### PR TITLE
perf: vs template support v3 http trigger notification bot

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep [hunk 1]
```diff
@@ -68,27 +85,18 @@ resource functionApp 'Microsoft.Web/sites@2021-02-01' = {
...
 
-output sku string = functionAppSKU
-output appName string = functionAppName
-output domain string = functionApp.properties.defaultHostName
-output appServicePlanName string = serverfarmsName
-output functionAppResourceId string = functionApp.id
-output functionEndpoint string = 'https://${functionApp.properties.defaultHostName}'
\ No newline at end of file
+output BOT_DOMAIN string = functionApp.properties.defaultHostName
+output APP_SERVICE_PLAN_NAME string = serverfarmsName
```
Comment (xzf0587) [inline templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep:99]: APP_SERVICE_PLAN_NAME  is useless now. Remove it from output

File: templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep [hunk 2]
```diff
@@ -68,27 +85,18 @@ resource functionApp 'Microsoft.Web/sites@2021-02-01' = {
...
+output FUNCTION_APP_RESOURCE_ID string = functionApp.id
```
Comment (xzf0587) [inline templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep:100]: to keep output key aligned, replace it with BOT_AZURE_FUNCTION_APP_RESOURCE_ID

File: templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep (hunk 1 above)
Comment (wenytang-ms) [reply templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep:99]: also remove serverfarm?

Comment (wenytang-ms) [reply templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep:99]: also remove the serverfarm  on Line 22?

File: templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep (hunk 2 above)
Comment (wenytang-ms) [reply templates/scenarios/notification-http-trigger-csharp/infra/azure.bicep:100]: sure

**Review by Siglud — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Pull request merged**